### DISCUSSION TITLE
Allow forks of own gists

### DISCUSSION THREAD

**Discussion by ackvf** [community/community#14882] [Other Feature Feedback, Questions, & Ideas]
It seems that people have been searching for a way to fork their own gists since beginning. https://stackoverflow.com/questions/14988168/how-to-fork-my-own-gist-on-github

They were using numerous hacks, even such as manipulating the html and adding the button themselves.
However, non of that seems to be possible any more. Even when using the Github API you get this response:

###### `https://api.github.com/gists/:gist_id/forks`
```
{
  "message": "You cannot fork your own gist.",
  "errors": [
    {
      "resource": "Gist",
      "field": "forks",
      "code": "unprocessable"
    }
  ],
  "documentation_url": "https://docs.github.com/v3/gists/#fork-a-gist"
}
```

https://docs.github.com/v3/gists/#fork-a-gist

Sure I can clone the gist and create new one, but that does not create that _Forked from abc/xyz_ link, which adds more context and is generally useful.
So I wonder, why was this disallowed in the first place? And can we have this allowed?

**Comment by pmbatty** [comment]
My company is using gists in a way where it would be very useful to be able to fork your own gist. I really don't understand why this is not allowed.
